platform/nservicebus.include.md

```diff
@@ -1,5 +1,68 @@
-NServiceBus, the heart of the system, is a messaging and workflow framework. It helps create distributed systems that are scalable, reliable, and easy to change. It supports various messaging patterns, encapsulates long-running business processes as [sagas](/nservicebus/sagas), and provides abstractions over several [queuing](/transports/) and [storage](/persistence/) technologies.
+NServiceBus is the heart of a distributed system and the Particular Service Platform. It helps create systems that are scalable, reliable, and flexible.
 
-Most queuing technologies try to guarantee 'at least once', or even 'exactly once', message delivery. They usually fall short of this promise. NServiceBus solves intermittent delivery problems by  automatically retrying messages using various strategies. If all else fails, it forwards messages to an error queue. From there, [ServiceControl](/servicecontrol/) and [ServicePulse](/serviceinsight/) expose the problems for human intervention.
+At its core, NServiceBus works by routing _messages_ between _endpoints_. [Messages](/nservicebus/concept-overview.md#message) are plain C# classes that contain meaningful data for the business process that is being modeled.
 
-NServiceBus is modular in design and highly extensible. There are many options to choose from for queueing, storage, serialization, logging, and more. Many elements of the system can be replaced with custom implementations for specific scenarios.
+```csharp
+public class ProcessOrder : IMessage
+{
+    public int OrderId { get; set; }
+}
+```
+
+[Endpoints](/nservicebus/concept-overview.md#endpoint) are logical entities that send and/or receive messages.
+
+```csharp
+// Sending endpoint
+await endpoint.Send(new ProcessOrder { OrderId = 15 });
+
+// Receiving endpoint
+public class ProcessOrderHandler : IHandleMessages<ProcessOrder>
+{
+    public async Task Handle(ProcessOrder messsage, IMessageHandlerContext context)
+    {
+        // Do something with ProcessOrder message
+    }
+}
+```
+
+Endpoints can be running in different processes, on different machines, even at different times. NServiceBus makes sure that each message reaches its intended destination and is processed.
+
+NServiceBus accomplishes this by providing an abstraction over [existing queuing technologies](/transports/). While it's possible to work directly with queuing systems, NServiceBus provides extra features to make applications more reliable and scalable.
+
+## Reliable
+
+NServiceBus offers different ways of ensuring information is not lost due to failures in a system. First, NServiceBus provides native transaction support (e.g. DTC) for the underlying queuing technologies that support it, as well as its own transaction guarantees through the implementation of the [Outbox pattern](/nservicebus/outbox).
```

Comment:
> e.g. DTC

This may not be the best thing to mention here. Ultimately, NServiceBus will use transactions appropriately, and in many cases they do not elevate to the DTC. E.g. when using a SQL DB for transport, persistence, and business data. Those that do elevate to the DTC would probably only involve MSMQ and SQLT, which is a scenario which is probably diminishing in importance.